v5/ooapi.v5.core/Models/ErrorResponse.cs

```diff
@@ -1,91 +1,91 @@
 using Newtonsoft.Json;
 using System.Runtime.Serialization;
 
-namespace ooapi.v5.Models
+namespace ooapi.v5.Models;
+
+/// <summary>
+/// A system message including the error code and an explanation
+/// </summary>
+[DataContract]
+public class ErrorResponse
 {
-    /// <summary>
-    /// A system message including the error code and an explanation
-    /// </summary>
-    [DataContract]
-    public class ErrorResponse
+
+    public ErrorResponse()
     {
 
-        public ErrorResponse()
-        {
+    }
 
-        }
 
-        public ErrorResponse(int statusCode, string detail = "")
+    /// <param name="statusCode"></param>
+    /// <param name="detail"></param>
+    public ErrorResponse(int statusCode, string detail = "")
+    {
+        Status = statusCode.ToString();
+        Detail = detail;
+        switch (statusCode)
```

Comment:
Zou wat mooier kunnen met een dicationary lookup (`FrozenDictionary` in .net 8 ;) ), maar wellicht scope creep.

Reply:
Ik laat dit voor nu op deze manier